.ci/scripts/install-aarch64-deps.sh

```diff
@@ -1,5 +1,7 @@
 #!/usr/bin/env bash
-find -type d -name node_modules -print -exec rm -rf '{}' \; || true
+if [ -n "$CI" ]; then
```

Comment:
why? don't we need to reinstall all dependencies in any case?

Reply:
Travis CI provide a per job caching system, so we can use it to avoid C++ node_modules to be recompiled each time.

Reply:
It represent 3min build time gain in each integration tests job (cross-test, redis, node version)